enclave-runner/src/loader.rs

```diff
@@ -136,20 +149,31 @@ impl<'a> EnclaveBuilder<'a> {
             miscselect: None,
             signature: None,
             usercall_ext : None,
+            load_and_sign: None,
+            hash_enclave: None,
         };
```

Comment:
Please make a new private method `fn new_with_source(EnclaveSource) -> EnclaveBuilder` that you can call in both `new` and `new_from_memory`